msteams-platform/resources/samesite-cookie-update.md

```diff
@@ -80,6 +80,43 @@ Android WebView is a Chrome system component that allows Android apps to display
 > * You must declare first party cookies as `SameSite=Lax` or `SameSite=Strict`, as appropriate.
 > * You must declare third party cookies as `SameSite=None; Secure`.
 
+## Third party cookies deprecation
+
+All third party cookies set in the top-level domain are blocked when that domain is embedded in an iframe. This third party cookie deprecation is starting to roll out for all major browsers, with Chrome currently at a 1% rollout and other major browsers like Firefox and Safari to follow.
+
+This deprecation impacts a common scenario in which the external app is rendered within Teams in various entry points like personal apps, channel tabs, and conversational tabs.
```

Comment:
Should we also mention all the affected modalities, "and across web, desktop, and mobile Teams clients" (pretty sure this is the case, but confirm w/eng)

Reply:
@parthiiita, could you please confirm this.